medical_pathology/demo/medical_pathology_category_demo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+
+    <record id="medical_pathology_category_1" model="medical.pathology.category">
```

Comment:
@bmw95 - Actually the current approach outlined in the guidelines is to put ``demo`` at the very end of the ID, but this is fairly arbitrary and putting it at the very beginning would probably work just as well in 99% of cases.

@lasley - Got it. Fair enough.